tomviz/Utilities.cxx

```diff
@@ -240,4 +252,56 @@ QString readInPythonScript(const QString &scriptName)
   return "raise IOError(\"Couldn't read script file\")\n";
 }
 
+void createCameraOrbit(vtkSMSourceProxy *data, vtkSMRenderViewProxy *renderView)
```

Comment:
This file is beginning to look like a dumping ground for 'misc'. I think we should just merge this as-is, but consider bringing some pieces out into their own more focused source files. I will be pulling the Python loading code into a separate class, I think the animation code would probably benefit from the same.